__tests__/__snapshots__/feeds.ts.snap

```diff
@@ -2496,11 +2496,33 @@ Object {
 }
 `;
 
+exports[`query sourceFeed should display return the right posts after the first page being a mix of pinned and unpinned posts 1`] = `
```

Comment:
The mix ones are the first page, not necessarily the result here which is the second page, but we wanted to see that it will return the correct second page which should be unpinned since the last item of the mixed first page is unpinned.